conda/environments/cuml_dev_cuda10.1.yml

```diff
@@ -21,7 +21,7 @@ dependencies:
 - faiss-proc=*=cuda
 - umap-learn
 - scikit-learn=0.23.1
-- treelite=0.92
+- treelite=0.93
```

Comment:
Maybe treelite >=0.92,<=0.93?
When we don't have a preference, it seems beneficial to be a little more flexible.

Reply:
We actually want to avoid 0.92 and use 0.93 exclusively, to avoid conflict.

Reply:
`0.93` is understood by conda as `0.93.*` so we should be good to use this and pick up any patch/hotfixes